book/escaping.md

```diff
@@ -16,4 +16,6 @@ Escape to external command:
 
 ## Windows Note
 
-When running an external command on Windows, nushell [used to](https://www.nushell.sh/blog/2022-08-16-nushell-0_67.html#windows-cmd-exe-changes-rgwood) use [Cmd.exe](https://docs.microsoft.com/en-us/windows-server/administration/windows-commands/cmd) to run the command, as a number of common commands on Windows are actually shell builtins and not available as separate executables. [Coming from CMD.EXE](coming_from_cmd.md) contains a list of these commands and how to map them to nushell native concepts.
+When running an external command on Windows,
```

Comment:
Seems like these 3 lines should be a paragraph instead of one concept on each line.

Reply:
@fdncred Given that it is Markdown, they should ultimately display as a single paragraph anyway, right?

Reply:
ok, sounds good. thanks for confirming. are you ok with these changes?

Reply:
I'm good - I don't have much to say from the Windows side, but I think we both agree that the doc shouldn't reference version numbers.